utilities/docker/geppetto-frontend/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM java:7

```

Comment:
@tarelli The original dockerfiles are using the gitall.py scripts to pull the geppetto source code with development as default. We can make that file take a parameter with the branch name where we will be pulling the code, but problem is unless there's a branch with that name for all bundles it will fail. This second dockerfile clones the geppetto gundles one by one, and uses the external parameter to pull the frontend from that parameter specified branch